frontend/src/libs/components/link/link.tsx

```diff
@@ -1,15 +1,30 @@
 import { NavLink } from 'react-router-dom';
 
 import { type AppRoute } from '#libs/enums/enums.js';
+import { mergeStyles } from '#libs/helpers/helpers.js';
 import { type ValueOf } from '#libs/types/types.js';
 
+import styles from './styles.module.scss';
+
 type Properties = {
   to: ValueOf<typeof AppRoute>;
   children: React.ReactNode;
+  className?: string | undefined;
 };
 
-const Link: React.FC<Properties> = ({ children, to }: Properties) => (
-  <NavLink to={to}>{children}</NavLink>
-);
+const Link: React.FC<Properties> = ({
+  children,
+  to,
+  className = '',
+}: Properties) => {
+  const { link } = styles;
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+  const mergedStyles = mergeStyles(className ?? '', link);
```

Comment:
Maybe className form props should to override the link one?

Reply:
yes it does. In helper function, the new styles are join to the end, so there is override. The idea is not to overwrite completely, but the repeated properties, that's why I put them at the end